web/wp-content/themes/dctx2/inc/template-tags.php

```diff
@@ -41,38 +41,38 @@ function dctx_posted_on() {
 
 		echo '<span class="posted-on">' . $posted_on . '</span><span class="byline"> ' . $byline . '</span>'; // WPCS: XSS OK.
```

Comment:
:warning: **Warning**: Using the WPCS native whitelist comments is deprecated. Please use the PHPCS native &quot;phpcs:ignore Standard.Category.SniffName.ErrorCode&quot; annotations instead. Found: // WPCS: XSS OK.\n (*WordPress.Security.EscapeOutput.DeprecatedWhitelistCommentFound*).